beacon/handler.go

```diff
@@ -114,7 +114,10 @@ func (h *Handler) ProcessPartialBeacon(c context.Context, p *proto.PartialBeacon
 	nextRound, _ := NextRound(h.conf.Clock.Now().Unix(), h.conf.Group.Period, h.conf.Group.GenesisTime)
 	currentRound := nextRound - 1
 
-	if p.GetRound() > currentRound {
+	// we allow one round off in the future because of small clock drifts
+	// possible, if a node receives a packet very fast just before his local
+	// clock passed to the next round
+	if p.GetRound() > nextRound {
```

Comment:
does this allow someone to run ahead too much? should it be bounded any more than this?

Reply:
It only allows a node to accept a partial signature at _one_ round ahead of the "present round" given by the local clock of the node.

Reply:
I've seen these debug messages a lot in the circleci output from the test-integration and it seems to work much more smoothly now.

Reply:
I'm running it a few times in a row to make sure.